### PR TITLE
feat/react-i18n Adding react-i18n for localization

### PR DESCRIPTION
**Pull request by fqlx**
feat/react-i18n Adding react-i18n for localization

### CODE REVIEW

File: shared/src/i18n/config.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export const resources = {
+  en: {
+    webapp,
+    shared,
+  },
+} as const;
+
+i18n.use(initReactI18next).init({
+  lng: "en",
+  ns: ["ns1", "ns2"],
```
Comment (leonkho) [inline shared/src/i18n/config.ts:15]: Should this be changed up?

Comment (fqlx) [reply shared/src/i18n/config.ts]: Fixed.

File: shared/src/i18n/en/webapp.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      "title": "STRIKE PRICE",
+      "explanation": "A strike price is the set price at which an option contract can be bought or sold when it is exercised.",
+      "learnMoreURL": "https://www.investopedia.com/terms/s/strikeprice.asp"
+    },
+    "Call": {
+      "explaination": "A call option is a financial contract that gives the option buyer the right but not the obligation to buy an asset at the strike price at a specific point in time.",
+      "learnMoreURL": "https://www.investopedia.com/terms/c/coveredcall.asp"
+    },
+    "Put": {
+      "explaination": "A put option is a derivative instrument which gives the holder the right to sell an asset, at a specified price, by a specified date to the writer of the put.",
```
Comment (kenchangh) [inline shared/src/i18n/en/webapp.json:13]: typo

**Review by cjinghong — APPROVED** [review]

File: shared/src/i18n/en/webapp.json (hunk 2 above)
Comment (fqlx) [reply shared/src/i18n/en/webapp.json]: fixed

**Review by cjinghong — APPROVED** [review]

**Pull request merged**